### ISSUE TITLE
Using QueryModel to find booleans

### ISSUE THREAD

**Issue by drmikecrowe** [gms1/node-sqlite3-orm#74]
I have a field defined as:

```ts
    @field({ name: 'tts_title_loaded' })
    ttsTitleLoaded: boolean = false;
```

I'm trying to search for  `ttsTitleLoaded == false`.

This works:

```ts
const podcastQuery = new QueryModel(Podcast);
const podcasts = await podcastQuery.selectAll(await this.sqldb, {
    where: {
        not: { ttsTitleLoaded: true },
    },
});
```
but this does not:

```ts
const podcastQuery = new QueryModel(Podcast);
const podcasts2 = await podcastQuery.selectAll(await this.sqldb, {
    where: {
        ttsTitleLoaded: false,
    },
});
```

Is that expected?

**Comment by gms1** [comment]
this issue seems to be specific to binding `false` value in the shorthand form.
this should be exactly the same, but works:
```
where: {
        ttsTitleLoaded: {eq: false},
    },
```

**Comment by gms1** [comment]
many thanks for this bug report;
this will be fixed in the next release

**Comment by gms1** [comment]
fixed in 2.4.18
